ksa_compliance/ksa_compliance/doctype/zatca_business_settings/zatca_business_settings.py

```diff
@@ -70,6 +75,14 @@ def after_insert(self):
         invoice_counting_doc.previous_invoice_hash = \
             "NWZlY2ViNjZmZmM4NmYzOGQ5NTI3ODZjNmQ2OTZjNzljMmRiYzIzOWRkNGU5MWI0NjcyOWQ3M2EyN2ZiNTdlOQ=="
         invoice_counting_doc.insert(ignore_permissions=True)
+        # Create Tax Account under Duties and Taxes Account
```

Comment:
Do we need this configuration before or after insert? It's all part of the same transaction, so failure at any point will abort the insertion. The advantage of putting it in before_insert is that we don't have to save again after inserting.